openapi_tester/schema_tester.py

```diff
@@ -198,6 +199,61 @@ def get_response_schema_section(self, response: Response) -> dict[str, Any]:
             )
         return {}
 
+    def get_request_body_schema_section(self, request: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Isn't this type wrong? We're passing in `response.request` further down, and I believe that's why you needed the `type: ignore`'s
```suggestion
    def get_request_body_schema_section(self, request: Request) -> dict[str, Any]:
```

Reply:
Indeed, that was the case :) set proper type and removed the unnecessary `# type: ignore`